src/sider/historikk/Historikk.tsx

```diff
@@ -43,6 +43,8 @@ function tagFromKilde(kilde: HistorikkEventType): ReactElement {
       return <Tag variant="alt3">Oppfølgingsplan</Tag>;
     case "OPPFOLGINGSPLAN_LPS":
       return <Tag variant="alt3">Oppfølgingsplan LPS</Tag>;
+    case "OPPFOLGINGSPLAN_FORESPORSEL":
+      return <Tag variant="alt3">Bedt om oppfølgingsplan</Tag>;
```

Comment:
`Forespørsel oppfølgingsplan`? De tagsene vi har nå er veldig lite "aktive" i ordlyden. Feks `Aktivitetskrav` i stedet for `Vurdert aktivitetskrav`, så hvis man skal holde seg på det nivået tenker jeg det kunne vært mer riktig å skrive noe sånt. Hva som "ble gjort" står kanskje i større grad i teksten?

Reply:
Ja, var det jeg tenkte på, vi kaller det forespørsel i selve action-knappen, så kan jo håpe at det er gjenkjennbart for veilederne.

Reply:
Mja, tenker det er greit å skille på forspørseler vs der hvor NAV har mottatt en oppfølgingsplan. 
Aktivitetskravet har jo enda flere potensielle "sub-typer" av hendelser så der gir det kanskje mer mening å bruke en overordnet tag, men det blir jo en avveining :man_shrugging: